scenarios/game/countdown.js

```diff
@@ -0,0 +1,22 @@
+import { store } from '../../src/game/game-store';
+import * as RoundPhaseActions from '../../src/shared/actions/round-phase-actions';
+import * as CountdownActions from '../../src/shared/actions/countdown-actions';
+import { DURATION } from '../../src/shared/constants/round';
+
+store.dispatch(RoundPhaseActions.startRound());
+
+async function pause(ms) {
```

Comment:
Is `async` required here?

Reply:
it is not _required_, as there is no `await` statement inside function body, however I consider this a good coding style. IMO everything that could be `await`-ed should be `async`